docs: adjusting for cleanup reference docs (#625)
openedx/docs.openedx.org: source/educators/references/resources_for_course_teams.rst

###################################
Resources for Open edX Course Teams
###################################

.. tags:: educator, reference

Course teams include faculty, instructional designers, course staff, discussion
moderators, and others who contribute to the creation and delivery of courses
on an Open edX site.

.. contents::
 :local:
 :depth: 1

Open edX Demo Course
====================

> The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to In discuss the tools and features that can leverage your learning program. After providing details about the platform, the course shows the different types of content that can be created, showcase the variety of assessment tools, and explore the different ways that you can engage your learners with social interactions.

Reviewer: ```suggestion
The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to discuss the tools and features that you can leverage in your learning program. After providing details about the platform, the course shows the different types of content that can be created, showcase the variety of assessment tools, and explore the different ways that you can engage your learners with social interactions.
```

Author: Done! Thanks for those edits!

> StudioX: Creating a Course with Open edX Studio

Reviewer: Sorry I wasn't specific enough in my previous comments that only the DemoX course exists. Please remove this, there is no Studio X course, or any of the following courses.

Author: done :)

===============================================

After you complete Open edX101, "StudioX" provides more detail about using Studio
to create a course, add different types of content, and configure your course
to provide an optimal online learning experience.

> BlendedX: Blended Learning with Open edX

Reviewer: Please remove this, there is no Blended X course

========================================

In "BlendedX" you explore ways to blend educational technology with
traditional classroom learning to improve educational outcomes.

> VideoX: Creating Video for the Open edX Platform

Reviewer: Please remove this, there is no Video X course

Author: done :)

###################################
Resources for Open edX Course Teams
###################################

.. tags:: educator, reference

Course teams include faculty, instructional designers, course staff, discussion
moderators, and others who contribute to the creation and delivery of courses

> on Open edX platform.

Reviewer: ```suggestion
on an Open edX site.
```

**********************************

> The Open edX Course Creator Series

Reviewer: There is no longer a course creator series, so just highlight the Demo course.

**********************************

The courses in the Open edX Course Creator Series provide foundational knowledge
about using the Open edX platform to deliver educational experiences. These courses
are available on Open edX platform.

.. contents::
 :local:
 :depth: 1

> Open edX101: Overview of Creating a Course

Reviewer: ```suggestion
Open edX Demo Course
```

We no longer have the "edX101" course. Please change this section to reference the Demo course and explain it briefly (look at the about page on https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about for more detail)

==========================================

> The `Open edX101 <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to provide a high-level overview of the course

Reviewer: ```suggestion
The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to provide a high-level overview of the course
```

**PDF**.

.. note:: If you use the Safari browser, be aware that it does not support the
 search feature for the HTML versions of the Open edX guides. This is a known
 limitation.

*****

> Email

Reviewer: Remove this section, it is entirely edx.org specific

Author: done!

**PDF**.

.. note:: If you use the Safari browser, be aware that it does not support the
 search feature for the HTML versions of the Open edX guides. This is a known
 limitation.

*****
Email
*****

To receive and share information by email, course team members can:

* Subscribe to announcements and other new topics in the Open edX Partner
  Portal or the Open edX Portal. For information about how to subscribe, see
  the :doc:`Release Announcements through the Open edX Portal <../../community/how-tos/receive_announcements_by_email>`

* Join the `openedx-studio`_ Google group to ask questions and participate in
  discussions with peers at other Open edX partner organizations and Open edX staffers.

*******************
Wikis and Web Sites
*******************

> The Open edX product team maintains public product roadmaps on `the Open edX Portal <https://openedx.org/>` and `the Open edX Commnity <https://openedx.org/community/>`_.

Reviewer: This is no longer accurate. Please reference the public roadmap at https://github.com/orgs/openedx/projects/4 and how to get involved with defining the roadmap by joining the Product Working Group https://openedx.atlassian.net/wiki/spaces/OEPM/overview

Author: done
